metar_taf_parser/parser/parser.py

```diff
@@ -311,7 +311,7 @@ def _extract_lines_tokens(self, taf_code: str):
                 lines_token[len(lines) - 1] = list(filter(lambda x: not x.startswith(TAFParser.TX) and not x.startswith(TAFParser.TN), last_line))
         return lines_token
 
-    def _parse_line(self, taf: TAF, line_tokens: list):
+    def _parse_line(self, taf: 'TAF', line_tokens: list):
```

Comment:
Because of the name collision, I wonder if it would help to import the `TAF` class under a different name.